pkgs/top-level/all-packages.nix

```diff
@@ -21277,10 +21277,7 @@ with pkgs;
 
   libpoly = callPackage ../applications/science/logic/poly {};
 
-  prooftree = (with ocaml-ng.ocamlPackages_4_01_0;
```

Comment:
Was there maybe a reason this specific version of ocaml was used?

Reply:
I suppose it was not compatible with our default, newer ocaml before that.